test/integration/requests/utility.ts

```diff
@@ -19,7 +20,7 @@ describe('Utility method integration tests', function () {
     const response = await (this.client as Client).request({
       command: 'ping',
     })
-    const expected = {
+    const expected: unknown = {
```

Comment:
why does this need to be typed?

Reply:
The `assert.deepEqual` is expecting the types to be the same, unless `unknown` or `any`